File: helium/src/main/groovy/com/stanfy/helium/handler/codegen/objectivec/ObjCImplementationFile.java
package com.stanfy.helium.handler.codegen.objectivec;

/**
 * Created by ptaykalo on 8/17/14.
 * Rperensents Implementation file (.m) with the Objective-C source
 */
public class ObjCImplementationFile extends ObjCFile {

  public ObjCImplementationFile(final String name) {
    super(name);
  }

  @Override
  public String getExtension() {
    return "m";
  }
}

File: codegen/objc/objc-entities/src/main/kotlin/com/stanfy/helium/handler/codegen/objectivec/entity/ObjCImplementationFile.kt
package com.stanfy.helium.handler.codegen.objectivec.entity;

import com.stanfy.helium.handler.codegen.objectivec.entity.builder.ObjCFile

/**
 * Created by ptaykalo on 8/17/14.
 * Rperensents Implementation file (.m) with the Objective-C source
 */
public class ObjCImplementationFile(name: String) : ObjCFile(name) {
  override fun getExtension(): String {
    return "m"
  }

}
